concept/type/impl/TypeImpl.java

```diff
@@ -272,28 +281,28 @@ public final boolean isDeleted() {
             return rpcTransaction;
         }
 
-        Stream<TypeImpl> stream(final TypeMethod.Iter.Req.Builder method, final Function<TypeMethod.Iter.Res, ConceptProto.Type> typeGetter) {
-            final TransactionProto.Transaction.Iter.Req request = TransactionProto.Transaction.Iter.Req.newBuilder()
-                    .setTypeMethodIterReq(method.setLabel(label)).build();
-            return rpcTransaction.iterate(request, res -> TypeImpl.of(typeGetter.apply(res.getConceptMethodTypeIterRes())));
+        Stream<TypeImpl> stream(final ConceptProto.Type.Req.Builder method, final Function<ConceptProto.Type.Res, ConceptProto.Type> typeGetter) {
+            final TransactionProto.Transaction.Req.Builder request = TransactionProto.Transaction.Req.newBuilder()
+                    .setTypeReq(method.setLabel(label));
+            return rpcTransaction.iterate(request, res -> TypeImpl.of(typeGetter.apply(res.getTypeRes())));
         }
 
-        Stream<ThingImpl> thingStream(final TypeMethod.Iter.Req.Builder method, final Function<TypeMethod.Iter.Res, ConceptProto.Thing> thingGetter) {
-            final TransactionProto.Transaction.Iter.Req request = TransactionProto.Transaction.Iter.Req.newBuilder()
-                    .setTypeMethodIterReq(method.setLabel(label)).build();
-            return rpcTransaction.iterate(request, res -> ThingImpl.of(thingGetter.apply(res.getConceptMethodTypeIterRes())));
+        Stream<ThingImpl> thingStream(final ConceptProto.Type.Req.Builder method, final Function<ConceptProto.Type.Res, ConceptProto.Thing> thingGetter) {
+            final TransactionProto.Transaction.Req.Builder request = TransactionProto.Transaction.Req.newBuilder()
+                    .setTypeReq(method.setLabel(label));
+            return rpcTransaction.iterate(request, res -> ThingImpl.of(thingGetter.apply(res.getTypeRes())));
         }
 
-        TypeMethod.Res execute(final TypeMethod.Req.Builder method) {
-            final TransactionProto.Transaction.Req request = TransactionProto.Transaction.Req.newBuilder()
-                    .setTypeMethodReq(method.setLabel(label)).build();
-            return rpcTransaction.execute(request).getTypeMethodRes();
+        ConceptProto.Type.Res execute(final ConceptProto.Type.Req.Builder method) {
+            final TransactionProto.Transaction.Req.Builder request = TransactionProto.Transaction.Req.newBuilder()
+                    .setTypeReq(method.setLabel(label));
+            return rpcTransaction.execute(request).getTypeRes();
         }
 
-        /*TypeMethod.Res iterate(final TypeMethod.Iter.Req.Builder method) {
-            final TransactionProto.Transaction.Iter.Req request = TransactionProto.Transaction.Iter.Req.newBuilder()
-                    .setTypeMethodIterReq(method.setLabel(label)).build();
-            return rpcTransaction.iterate(request).
+        /*ConceptProto.Type.Res iterate(final ConceptProto.Type.Req.Builder method) {
```

Comment:
meant to be commented out?

Reply:
I originally left it in there because it seemed likely to be needed in the future - but in fact, it actually doesn't and I've now deleted it.